### DISCUSSION TITLE
use vue3-carousel-nuxt

### DISCUSSION THREAD

**Discussion by hoseiniseyedali** [nuxt/nuxt#26826] [General]
I tried to use vue3-carousel-nuxt for my nuxyjs project, But my efforts were useless and it was not possible to use it and it returned the following error. Does anyone have experience using this?
error: Importing directly from module entry-points is not allowed.

**Comment by oritwoen** [comment]
Have you checked `@nuxt/ui`? It offers official and "native" support for such a component. I'll give you the link below.

https://ui.nuxt.com/components/carousel

If you want to use `vue3-carousel-nuxt`, please provide an example of your implementation code or a reproduction because it's hard to tell what went wrong.
